### PR TITLE
Add breadcrumb logic and styling

### LINKED ISSUE #7: [Site] Implement bread crumb
All pages, except homepage, use a breadcrumb.

- [ ] implement the breadcrumb as part of the header nav
- [ ] exclude breadcrumb on homepage
- [ ] convert german "umlauts" like ae > ä, oe > ö. ue >ü

Hint we go with a simple solution at the beginning and derive the breadcrumb segments from the URL, only the current page segment is used from the page title. 
A more advanced solution using index lookup etc will be built in a later iteration.

### PR DESCRIPTION
**Pull request by steinkopffp**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #7

Test URLs:
- Before: https://main--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen
- After: https://breadcrumb--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen

### CODE REVIEW

**Comment by mhaack** [comment]
The breadcrumb breaks the layout for normal pages with sidebar like https://breadcrumb--eder-group--hlxsites.hlx.page/newstermine/social-media or any news page

**Comment by mtilburgadobe** [comment]
@steinkopffp - looks like the breadcrumb is getting added after the Hero image, which is strange because I see you have the buildBreadcrumb before the buildHeroBlock in scripts.js, in any case, on this page you can see what I mean:

https://breadcrumb--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/detail/kundenmagazin-der-eder-weihnachtspost-2022

**Comment by alexcarol** [comment]
Also on this page: https://breadcrumb--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen if we click "news/termine" we get a 404, rather than redirecting to the same page we're in. Again, not sure if it's in the scope of this PR, but it should either be fixed or a new issue to implement a redirect should be created.

**Comment by mhaack** [comment]
> Also on this page: https://breadcrumb--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen if we click "news/termine" we get a 404
Fixed with setting up the correct redirect.

**Comment by mhaack** [comment]
Please double check on some news page:

Screenshot 2023-01-31 at 16 24 20

Also content, breadcrumb and logo are not aligned but this maybe should be done via separate PR as this is a wider issue.

File: blocks/breadcrumb/breadcrumb.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { getMetadata } from '../../scripts/lib-franklin.js';
+
+export default function decorate(block) {
+  if (block.children.length === 1) {
+    block.children[0].remove();
+    return;
+  }
+  const configRows = [...block.children];
+  const homeEntry = document.createElement('a');
+  const isPressemeldungen = getMetadata('template') === 'Pressemeldungen';
```
Comment (mhaack) [inline blocks/breadcrumb/breadcrumb.js:10]: Can't we have this more generic, without depending on a template? We might have similar logic for other templates like events or job posting.

Comment (alexcarol) [reply blocks/breadcrumb/breadcrumb.js:10]: Here's some options of what we could change. Would you implement any of these?
Options:
1. Rename variable to "ignoreDetail" - viable
2. Ignore all "detail" path components - risky (we might ignore something that we don't want to)
3. Add extra metadata to the content that we use to decide whether to ignore detail - expensive to implement and maintain by authors (updating import + code)
4. Wait until we have at least two cases and abstract appropriately - viable

Comment (mhaack) [reply blocks/breadcrumb/breadcrumb.js:10]: We already have:
* https://main--eder-karriere--hlxsites.hlx.live/bewerbung/offene-stellen/detail/vertriebsmitarbeiter-feedstar-mwd-verschiedene-gebiete
* https://main--eder-group--hlxsites.hlx.page/newstermine/veranstaltungen/detail/tier-technik-st-gallen
Both with detail in the path which we need to ignore.

Alternative is to remove `detail` from the URL, but then we break with old site URLs and would need tons of redirects setup.

**Review by mhaack — APPROVED** [review]

**Pull request merged**